src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -44,16 +82,56 @@ public static void PrintProcessStatus(IConsole console)
                     {
                         continue;
                     }
-
                     try
                     {
-                        sb.Append($"{process.Id, 10} {process.ProcessName, -10} {process.MainModule.FileName}\n");
+                        if(verbose)
+                        {
+                            int iDLength = 10;
+                            int nameLength = 10;
+                            int fileLength = 25;
+                            int cmdLength = 55;
+                            String cmdLineArgs = GetArgs(process);
+                            cmdLineArgs = cmdLineArgs == process.MainModule?.FileName ? "" : cmdLineArgs;
+                            string fileName = process.MainModule?.FileName ?? "";
+                            string[] cmdList = cmdLineArgs.Split(" ");
+                            foreach(string str in cmdList)
+                            {
+                                string splitChar = "";
+                                if (str.Contains("/"))
+                                {
+                                    splitChar = "/";
+                                    
+                                }
+                                else if(str.Contains("\\"))
+                                {
+                                    splitChar = "\\";
+                                }
```

Comment:
I would use [`Path.PathSeperator`](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.pathseparator?view=net-6.0) here. Alternatively, you could determine if you're on a non-Windows platform and change behavior based on that.

Reply:
```suggestion
                                char splitChar = str.Contains(Path.PathSeperator) ? Path.PathSeperator : '\0';
```